tests/acceptance/features/bootstrap/SpacesContext.php

```diff
@@ -570,45 +570,15 @@ public function theUserLooksUpTheSingleSpaceUsingTheGraphApiByUsingItsId(string
 		);
 	}
 
-	/**
-	 * @When /^user "([^"]*)" (?:creates|tries to create) a space "([^"]*)" of type "([^"]*)" with the default quota using the Graph API$/
-	 *
-	 * @param string $user
-	 * @param string $spaceName
-	 * @param string $spaceType
-	 *
-	 * @return void
-	 *
-	 * @throws GuzzleException
-	 * @throws Exception
-	 */
-	public function theUserCreatesASpaceUsingTheGraphApi(
-		string $user,
-		string $spaceName,
-		string $spaceType
-	): void {
-		$space = ["Name" => $spaceName, "driveType" => $spaceType];
-		$body = json_encode($space, JSON_THROW_ON_ERROR);
-		$this->featureContext->setResponse(
-			GraphHelper::createSpace(
-				$this->featureContext->getBaseUrl(),
-				$user,
-				$this->featureContext->getPasswordForUser($user),
-				$body
-			)
-		);
-
-		$this->setSpaceCreator($spaceName, $user);
-	}
-
 	/**
 	 * @When /^user "([^"]*)" creates a space "([^"]*)" of type "([^"]*)" with quota "([^"]*)" using the Graph API$/
 	 * @When /^user "([^"]*)" tries to create a space "([^"]*)" of type "([^"]*)" with quota "([^"]*)" using the Graph API$/
```

Comment:
```suggestion
	 * @When /^user "([^"]*)" (?:creates|tries to create) a space "([^"]*)" of type "([^"]*)" with quota "([^"]*)" using the Graph API$/
```
Can we change here too?